JavaScript/math/gcd_of_two_no.js

```diff
@@ -0,0 +1,37 @@
+/*
+A program to find Greatest Common Divisor(GCD) of two Numbers.
+GCD of two integers is the largest positive integer that divides each of the integers.
+*/
+
+var number1 = 10;
+var number2 = 15;
```

Comment:
Use Dynamic Input

Reply:
what should i use prompt or something else

Reply:
Take the input from the User.

Reply:
Done